packages/ag-charts-enterprise/src/features/annotations/annotationOptionsToolbar.ts

```diff
@@ -387,8 +387,6 @@ export class AnnotationOptionsToolbar extends _ModuleSupport.BaseProperties {
     }) {
         const { buttonBounds, popoverBounds } = event;
 
-        this.hideOverlays();
```

Comment:
This was a carry-over from before the refactor, it's not needed here as the menus and dialog are already closed by clicking elsewhere.